packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -361,18 +380,23 @@ async function launchDesktopApp(app: OfficeApp, manifest: ManifestInfo, document
     throw new ExpectedError(`Sideload to the ${getOfficeAppName(app)} app is not supported.`);
   }
 
-  // for Outlook, open Outlook.exe; for other Office apps, open the document
+  // for Outlook, Word, Excel, PowerPoint open {Host}.exe; for other Office apps, open the document
+  let path: string;
   if (app == OfficeApp.Outlook) {
     const version: string | undefined = await getOutlookVersion();
     if (version && !hasOfficeVersion("16.0.13709", version)) {
       throw new ExpectedError(
         `The current version of Outlook does not support sideload. Please use version 16.0.13709 or greater.`
       );
     }
-    await launchApp(app, await getOutlookExePath());
+    path = await getOfficeExePath(app);
+  } else if (manifest.manifestType === ManifestType.JSON) {
+    path = await getOfficeExePath(app);
```

Comment:
Is this changing things to launch Word/Excel/PPT apps based on the exe path rather than opening a document with a reference to the add-in?

Reply:
For XML add-ins we don't change the launching pattern in our changes, it is still a document with add-in embedded via web-extension.

Reply:
That doesn't answer the question of do we need to change the WXP launching pattern at all (even for json)?